src/ErrorModal.svelte

```diff
@@ -20,7 +21,7 @@
 </script>
 
 <Modal on:close error {floating} {subtle}>
-  <span slot="title">
+  <span slot="title" use:twemoji>
```

Comment:
This could be moved to the inner `div`.

Reply:
Yep we could, the thing is that sometimes `title` can be an emoji too..